README.md

```diff
@@ -12,6 +12,35 @@ Swagger dokumentácia pre HTTP API je [dostupná na githube](https://generator3.
 
 Vyvolať spustenie programu je možné priamo z webového prehliadača otvorením adresy so špeciálnym protokolom `autogram://`. Napríklad cez `autogram://go`.
 
+## Štátne elektronické formuláre
```

Comment:
Toto by som dal az pod sekciu podporovane karty, je to extremny detail.

Reply:
upravil som